modules/core/src/main/java/org/apache/synapse/config/xml/SequenceMediatorFactory.java

```diff
@@ -69,6 +69,7 @@ public SequenceMediator createAnonymousSequence(OMElement elem, Properties prope
         }
         addChildren(elem, seqMediator, properties);
         seqMediator.setSequenceType(SequenceType.ANON);
+
```

Comment:
Unnecessary new line?

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/1439

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/1439